clojure/README.md

```diff
@@ -0,0 +1,43 @@
+# clojure
+
+Clojure is a robust, practical, and fast programming language with a set of useful features that together form a simple, coherent, and powerful tool.
+
+## Maintainers
+
+* The Habitat Maintainers: <humans@habitat.sh>
+
+## Type of Package
+
+Binary package
```

Comment:
This seems to provide `clj` as a binary but also nrepl server as a service - perhaps both should be declared here? Or if a plan provides both it should be called a Service package.

Reply:
Good catch.

This should be declared as a service package.

Reply:
@predominant: have a look now